packages/store-cache/package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "@latticexyz/store-cache",
+  "version": "1.42.0",
+  "private": true,
+  "description": "Database to replicate Store contract state on a TypeScript client",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/latticexyz/mud.git",
+    "directory": "packages/store-cache"
+  },
+  "license": "MIT",
+  "type": "module",
+  "exports": {
+    ".": "./dist/index.js"
+  },
+  "types": "src/index.ts",
+  "scripts": {
+    "build": "tsup",
+    "clean": "rimraf dist",
```

Comment:
not blocking and not at all critical but I've been trying to keep the build scripts somewhat aligned so if we ever want to revisit turbo build steps, etc., we don't have to go through and audit all the build scripts again